### ISSUE TITLE
Question for the visualization of the learned feature as  Fig.7

### ISSUE THREAD

**Issue by fuyimin96** [iCGY96/ARPL#6]
To get the visualization as in the paper, you use t-sne or use a fc(out_channel=2) to do that?
Thanks

**Comment by fuyimin96** [comment]
Chiese version: 如文中图7，12特征可视化，是对高维embedding用tsne降维；还是设置输出维度为2的全连接层得到；还是选取了高维rpl特征中的两个维度进行可视化？
谢谢！

**Comment by iCGY96** [comment]
fc(out_channel=2)

**Comment by fuyimin96** [comment]
So,  the fc layer is add behind the rpl (dim=latent_size,e.g., 128), or you use the fc layer to the last feature map to form the rpl(latent_size=2)
Thanks

**Comment by iCGY96** [comment]
For feature visualization, both feature and RP are set to dimension 2.

For details of the experiment, see the repo below:
https://github.com/KaiyangZhou/pytorch-center-loss

I hope it can help you.

**Comment by fuyimin96** [comment]
Thanks, I sincerely got your help. As I think that t-sne will break the feature representation.
😊

**Comment by fuyimin96** [comment]
I have sent the agreement to  pkuml@pku.edu.cn, have you received it?
Thanks

**Comment by fuyimin96** [comment]
@iCGY96

**Comment by rameshshreyas** [comment]
@iCGY96 @fuyimin96  Do you train the final fc layer to make the visualizations?
